backend/canisters/user_index/impl/src/updates/set_username.rs

```diff
@@ -76,8 +76,16 @@ pub fn validate_username(username: &str) -> UsernameValidationResult {
         return UsernameValidationResult::Invalid;
     }
 
-    if username.replace('_', "").to_uppercase() == "OPENCHATBOT" {
-        return UsernameValidationResult::Invalid;
+    let normalised = username.replace('_', "").to_uppercase();
```

Comment:
Nit - I would put this in a separate function something like isUsernameReserved

Reply:
Done